index.js

```diff
@@ -77,9 +73,44 @@ class BookLibrary {
   }
 
   clearForm() {
-    this.bookTitle.value = "";
-    this.bookAuthor.value = "";
+    this.bookTitle.value = '';
+    this.bookAuthor.value = '';
   }
 }
 // eslint-disable-next-line no-unused-vars
 const library = new BookLibrary();
+
+const currentDate = new Date();
+const options = {
+  year: 'numeric', month: 'long', day: 'numeric', hour: 'numeric', minute: 'numeric', second: 'numeric', hour12: false,
+};
+const formattedDate = currentDate.toLocaleDateString('en-US', options);
+const displayDate = document.getElementById('date-display');
+displayDate.textContent = formattedDate;
+
+document.addEventListener('DOMContentLoaded', () => {
+  const navItems = document.querySelectorAll('.nav-item');
+  const contentSections = document.querySelectorAll('.content-section');
+
+  navItems.forEach((item) => {
+    item.addEventListener('click', (event) => {
+      event.preventDefault();
+
+      const targetSectionId = event.currentTarget.getAttribute('data-target');
+      // eslint-disable-next-line no-use-before-define
```

Comment:
- [ ] Please avoid disabling linter rules, they serve as a way of having a consistent code-base and ensuring best practices! Instead, please consider solving the issues. 🤓 🔧